arviz/tests/base_tests/test_plot_utils.py

```diff
@@ -264,3 +265,25 @@ def test_dealiase_sel_kwargs():
     assert res["alpha"] == 0.4
     assert "line_color" in res
     assert res["line_color"] == "red"
+
+
+# Check if Bokeh is installed
+bokeh_installed = importlib.util.find_spec("bokeh") is not None  # pylint: disable=invalid-name
+pytest.mark.skipif(
+    not (bokeh_installed | running_on_ci()), reason="test requires bokeh which is not installed",
+)
+
+
+def test_set_bokeh_circular_ticks_labels():
+    import bokeh.plotting as bkp
+
+    """Assert the axes returned after placing ticks and tick labels for circular plots."""
```

Comment:
```suggestion
@pytest.mark.skipif(
    not (bokeh_installed | running_on_ci()), reason="test requires bokeh which is not installed",
)
def test_set_bokeh_circular_ticks_labels():
    """Assert the axes returned after placing ticks and tick labels for circular plots."""
    import bokeh.plotting as bkp

```

I think this will work and will skip only this test instead of the whole file. I have also moved the docstring back to below the function definition